tests/contrib/django/test_connection.py

```diff
@@ -30,7 +26,7 @@ def test_connection(self):
 
         span = spans[0]
         eq_(span.name, 'sqlite.query')
-        eq_(span.service, 'defaultdb')
+        eq_(span.service, 'django-defaultdb')
```

Comment:
Yes a prefix is interesting, since having the DB alias name in the service name is something we should keep. Let's say that we may:
```python
# your Django settings.py
APM_SERVICE = 'my-django'
DATADOG_TRACE = {
    'DEFAULT_SERVICE': APM_SERVICE,
    'DEFAULT_DATABASE_PREFIX': APM_SERVICE,
}
```
this will end having the following services:
* `my-django`: it's your Django application
* `my-django-defaultdb`: it's your main Django database (R/W)
* `my-django-read-replica`: it's your replica DB

By default `DEFAULT_DATABASE_PREFIX` is empty so we don't "break" current applications stats. Sounds good to you?